modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to:
+
+* Release the persistent volume from the persistent volume claim.
+* Erase all data in the share.
+* Create a new persistent volume with a different name,
+maintaining the persistent volume's original connection settings.
+
+.Prerequisite
+* Note the connection details of the persistent volume you want to reclaim.
```

Comment:
Is the user going to already know how to do this, or is there a separate module that you could include that describes how to make this happen?

Reply:
I added: 

Use the Web Console or command line to display storage information for the 
virtual machine.

Reply:
I don't think this is enough information. Move this to the procedure and add an explicit step for at least one of these methods.

Reply:
. Enter the following command for the persistent volumes
you want to reclaim. Save the output for future reference.
+
----
$ oc describe pv <pv_name>
----